server/main.go

```diff
@@ -3,13 +3,15 @@ package main
 import (
 	"log"
 	"net/http"
+
+	gitop "github.com/litmuschaos/charthub.litmuschaos.io/server/pkg/gitops"
+	routes "github.com/litmuschaos/charthub.litmuschaos.io/server/routes"
 )
 
 func main() {
 
 	// Trigger is go-routine which synchronously calls the git-ops function Trigger()
-	go Trigger()
-
-	router := NewRouter()
+	go gitop.Trigger()
```

Comment:
Add the comment for this function

Reply:
I think the changes are not showing here... there is a comment already there...